mesa/experimental/devs/simulator.py

```diff
@@ -164,7 +157,7 @@ def schedule_event_relative(
 
         """
         event = SimulationEvent(
-            self.time + time_delta,
+            self.model._time + time_delta,
```

Comment:
perhaps betraying my java background, but changing attributes of other objects directly like this makes me uncomfortable.

Reply:
I understand that. I think in this case it allows the model to be the ground truth of time. We should document it as "A DEVS simulator is an alternative way to manage agent activation and progress time".